### ISSUE TITLE
Error mode persists after clearing

### ISSUE THREAD

**Issue by fallingcats** [vermut/ha_openmower#3]
The main mower entity seems to stay in error mode even after an emergency is cleared, and even after the openmower container is restarted.

![openmower stating in error in HA](https://github.com/vermut/ha_openmower/assets/12884838/9e7fe9d1-9692-4005-880b-54340fb01d8d)

**Comment by vermut** [comment]
This is a logic that I've copied from someone's code. It says that mower is only healthy if it's mowing or docked. If it's idle outside the dock - this is considered an error.

Not sure if it's too much overthinking.

(I'm assuming that is your situation - mower idle outside the dock, right?)

**Comment by fallingcats** [comment]
Yeah you're right. Now I'm wondering if docking doesn't work in "Error (idle)" mode because Home Assistant prevents it, or if openmower generally just can't start docking while idle.

**Comment by vermut** [comment]
I'm not sure if HA prevents buttons, but maybe it does. Easy test would be clicking dock button in OM Web App - if it's working and HA not - that would be a bug.

**Comment by fallingcats** [comment]
I don't think there is a button anywhere to start docking when idle, which is why I can't test where it fails exactly.
